### ISSUE TITLE
can't reach mysql container from prestashop container

### ISSUE THREAD

**Issue by KannarFr** [PrestaShop/docker#12]
hi,

i ran
`docker run -p 3306:3306 --name mysql -v ~/datadir/:/var/lib/mysql/ -e MYSQL_ROOT_PASSWORD=superpass -d mysql:latest`
and 
`docker run -ti --name prestashop -p 8080:80 -d prestashop/prestashop`

then in the prestashop installation wizard i can't reach the database, how can i create the link between these two containers?

Alex :)

**Comment by Quetzacoalt91** [comment]
Hi,

When running your second container with PrestaShop inside, use the parameter `--link`.
In your case, you will get:
```
 docker run -ti --name prestashop --link mysql -p 8080:80 -d prestashop/prestashop
```

In the installation process, set the MySQL address as `mysql`.

Best regards

**Comment by KannarFr** [comment]
The database hostname as mysql? That's what you say?

Thanks, i'll try :)

**Comment by Quetzacoalt91** [comment]
Yep, exactly.

**Comment by KannarFr** [comment]
It's works! Thanks.

**Comment by KannarFr** [comment]
Which is the best database for your image/prestashop, i mean i got an sql error, 

`All parts of a PRIMARY KEY must be NOT NULL; if you need NULL in a key, use UNIQUE instead`

**Comment by Quetzacoalt91** [comment]
I advise you to use the container mysql:5.6 :)

**Comment by KannarFr** [comment]
It has been work until... Haven't details on error :s suggestion?
![sugg](https://cloud.githubusercontent.com/assets/7518245/14387887/b07f7236-fdab-11e5-91d4-ee56226abe3a.png)

**Comment by KannarFr** [comment]
Ok it works until a new error, without any specifications.. Still at 12%, i read on some prestashop forum that's can be due to a timeout while installation in php.ini, in your docker image did you specify something in php.ini?

**Comment by KannarFr** [comment]
I changed it to 9999, but still got error... :/

**Comment by KannarFr** [comment]
Ok i solved this, by using sql:5.5. Found this solution on last post at:
https://www.prestashop.com/forums/topic/294168-create-database-tables-an-error-occurred-during-installation/page-2
